### PR TITLE
Update 2.1 Branch Version

### PR DESCRIPTION
**Pull request by lindluni**
Signed-off-by: Brett Logan <brett.t.logan@ibm.com>

### CODE REVIEW

**Review by suryalnvs — DISMISSED** [review]

File: ci/scripts/build/fabric-chaincode-java.sh [hunk 1]
```diff
@@ -9,4 +9,4 @@ cd "${GOPATH}/src/github.com/hyperledger/fabric-chaincode-java"
 ./gradlew buildImage -x test
 
 # This is temporary until chaincode-java starts adding 2 digit tags
-docker tag hyperledger/fabric-javaenv hyperledger/fabric-javaenv:2.1
+docker tag hyperledger/fabric-javaenv hyperledger/fabric-javaenv:latest
```
Comment (suryalnvs) [inline ci/scripts/build/fabric-chaincode-java.sh]: we don't need this as if tagging latest to latest again

File: ci/scripts/build/fabric-chaincode-node.sh [hunk 2]
```diff
@@ -10,4 +10,4 @@ node common/scripts/install-run-rush.js install
 node common/scripts/install-run-rush.js rebuild
 
 # This is temporary until chaincode-node starts adding 2 digit tags
-docker tag hyperledger/fabric-nodeenv hyperledger/fabric-nodeenv:2.1
+docker tag hyperledger/fabric-nodeenv hyperledger/fabric-nodeenv:latest
```
Comment (suryalnvs) [inline ci/scripts/build/fabric-chaincode-node.sh]: same as above

Comment (lindluni) [reply ci/scripts/build/fabric-chaincode-node.sh]: I feel like maybe I lost my mind today

**Comment by lindluni** [comment]
Fixed

File: ci/scripts/build/fabric-chaincode-node.sh (hunk 2 above)
Comment (ryjones) [reply ci/scripts/build/fabric-chaincode-node.sh]: just today?

**Review by suryalnvs — APPROVED** [review]

**Pull request merged**